modules/nlu/src/views/entities/editor.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import style from './style.scss'
+import { ListGroupItem, Glyphicon } from 'react-bootstrap'
+import _ from 'lodash'
+import { WithContext as ReactTags } from 'react-tag-input'
+import classNames from 'classnames'
+
+export default class EntityEditor extends React.Component {
+  constructor(props) {
+    super(props)
+    this.occurenceInputRef = React.createRef()
+  }
+
+  state = {
+    currentOccurence: undefined,
+    currentEntity: undefined,
+    occurenceInput: ''
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.entity !== this.state.currentEntity) {
+      const newEntity = nextProps.entity
+
+      this.setState({ currentEntity: newEntity, currentOccurence: newEntity && newEntity.occurences[0] })
+    }
+  }
+
+  onEnterPressed = (event, cb) => {
+    const enterKey = 13
+    if (event.keyCode === enterKey) {
+      cb()
+    }
+  }
```

Comment:
Should be called handleKeyPress and could be shorter: event.key=== 'Enter' && cb()